cypress/e2e/teams/studentCreateEditDeleteTeam.feature

```diff
@@ -1,44 +1,56 @@
-@unstable_test
+@stable_test
+@release
 Feature: Team - Student managed teams (on BRB)
```

Comment:
remove `(on BRB)`

Reply:
on NBC, its not possible for student to create a team, admin does not have this checkbox either.
so this feature can be for BRB & dBC.